### ISSUE TITLE
Is it possible to change select button to other button?

### ISSUE THREAD

**Issue by LuisCarlosHK** [TheOfficialFloW/VitaShell#551]
My select button is not working.
Is there anyway to change the select button to other button?

**Comment by theheroGAC** [comment]
yes here https://github.com/theheroGAC/VitaShell/releases/tag/v2.00U

**Comment by LuisCarlosHK** [comment]
> yes here https://github.com/theheroGAC/VitaShell/releases/tag/v2.00U

I can't use vitashell transfer files. How can I transfer files without vitashell?

**Comment by theheroGAC** [comment]
Use filezilla?

**Comment by LuisCarlosHK** [comment]
> Use filezilla?

My button is not working so I can't use vitashell to transfer files.

**Comment by theheroGAC** [comment]
> > Use filezilla?
> 
> My select button is not working so I can't use vitashell to transfer files.

Use this vitashell.https://github.com/theheroGAC/VitaShell/releases/download/v2.00U/VitaShell.vpk
replaces the select button on vitashell with LTRIGGER

**Comment by LuisCarlosHK** [comment]
> > > Use filezilla?
> > 
> > 
> > My select button is not working so I can't use vitashell to transfer files.
> 
> Use this vitashell.https://github.com/theheroGAC/VitaShell/releases/download/v2.00U/VitaShell.vpk
> replaces the select button on vitashell with LTRIGGER
> Use QR-CODE for download here https://store.brewology.com/vita/ahomebrew.php?brewid=317

Finally, I rename the vpk to mp4 and use qcma copy the file to psv, then rename it back to vpk in psv and install, but got error 0x80101114.

**Comment by theheroGAC** [comment]
> > > > Use filezilla?
> > > 
> > > 
> > > My select button is not working so I can't use vitashell to transfer files.
> > 
> > 
> > Use this vitashell.https://github.com/theheroGAC/VitaShell/releases/download/v2.00U/VitaShell.vpk
> > replaces the select button on vitashell with LTRIGGER
> > Use QR-CODE for download here https://store.brewology.com/vita/ahomebrew.php?brewid=317
> 
> Finally, I rename the vpk to mp4 and use qcma copy the file to psv, then rename it back to vpk in psv and install, but got error 0x80101114.

Test this https://store.brewology.com/vita/ahomebrew.php?brewid=538

**Comment by rgoulart23** [comment]
@theheroGAC just wanted to say thank you for this. My select button stopped working and I found this git page from google. I downloaded your version, changed it to .mp3 and transfered to Vita using Content Manager. Changed it again to .vpk but it wouldn't let me install through VitaShell. So I changed it to .zip and extracted the eboot.bin file to the VitaShell folder, replacing the file there. After this I closed VitaShell and opened it again, and it was already your version. I can finally use VitaShell again using the L Trigger button. So thank you once more!

**Comment by alexander-pv** [comment]
> 
> 
> @theheroGAC just wanted to say thank you for this. My select button stopped working and I found this git page from google. I downloaded your version, changed it to .mp3 and transfered to Vita using Content Manager. Changed it again to .vpk but it wouldn't let me install through VitaShell. So I changed it to .zip and extracted the eboot.bin file to the VitaShell folder, replacing the file there. After this I closed VitaShell and opened it again, and it was already your version. I can finally use VitaShell again using the L Trigger button. So thank you once more!

Hi, I faced a similar issue with VitaShell 2.02. A manual copy of VitaShell 2.0 is working for me too. Thanks for the comment!
